imports/ui/genetable/columns/AttributeValue.jsx

```diff
@@ -4,6 +4,47 @@ function isArray(x) {
   return Array.isArray(x) && x.length > 1;
 }
 
+function addExtUrl(value, setDescription) {
```

Comment:
I don't think this has to be a separate function. By moving this logic to `DetailedSingleAttribute`, you can just have that component return the html anchor or plain value.